source/features/fork-source-link-same-view.tsx

```diff
@@ -8,36 +8,43 @@ import getDefaultBranch from '../github-helpers/get-default-branch';
 import {getRepo, getForkedRepo} from '../github-helpers';
 
 const isFilePath = (): boolean => pageDetect.isSingleFile()
-	|| (pageDetect.isRepoTree() && !pageDetect.isRepoRoot())
```

Comment:
This condition is what broke https://github.com/refined-github/refined-github/issues/5479 since it causes the root to not be 404-checked, I think